FirebaseVertexAI-Docs.podspec

```diff
@@ -0,0 +1,53 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseVertexAI'
+  s.version          = '0.1.0'
```

Comment:
Not sure if it matters for docs purposes but should this be `10.26.0`?

Reply:
Done.